cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -9,11 +9,13 @@
     "MasterInstanceType": {
       "Description": "Head node EC2 instance type",
       "Type": "String",
+      "Default": "t2.micro",
       "ConstraintDescription": "Must be a valid EC2 instance type, with support for HVM."
     },
     "ComputeInstanceType": {
       "Description": "ComputeFleet EC2 instance type",
       "Type": "String",
+      "Default": "t2.micro",
```

Comment:
We can improve the description by highlighting the fact that this parameter is useless for Slurm but needed as CFN param.

We can use NONE instead of t2.micro to avoid hiding other errors. In this way we'll be sure that a value is always passed.